x-pack/plugin/core/template-resources/src/main/resources/entsearch/connector/chat/comments-template.json

```diff
@@ -0,0 +1,46 @@
+{
+  "template": {
+    "mappings": {
+      "properties": {
+        "comments": {
```

Comment:
One thing that I conveniently skipped in the POC is attachments... comments can have attachments, for sure in sources like github and salesforce

Can we add minimal template for attachment, have a list of attachments in the comment?

Reply:
yeah good call. We probably need attachments as a top-level field, too.